presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -36,6 +36,14 @@ constexpr folly::StringPiece kCounterNumHTTPRequestError{
 constexpr folly::StringPiece kCounterHTTPRequestLatencyMs{
     "presto_cpp.http_request_latency_ms"};
 
+constexpr folly::StringPiece kCounterHttpClientPrestoExchangeNumOfOnBody{
```

Comment:
```
/// The number of received http buffers from presto exchange source.
s/kCounterHttpClientPrestoExchangeNumOfOnBody/kCounterHttpClientPrestoExchangeNumReceivedBuffers/
s/presto_exchange.num_of_onBody/presto_exchange.num_received_buffers/
```

Reply:
I understand where you're coming from, but I'm not sure about the renaming because we want to expose the http related counters, and `num_of_onBody` indicates it's within the https proxygen framework; if we use `num_received_buffers` then it more likely indicates it's related to presto's own buffer logic

Recently Sergey was debugging a local exchange issue and was also confused by the naming, so I will leave it as it is for debugging purpose